app/assets/javascripts/application.js

```diff
@@ -1,5 +1,3 @@
-//= require jquery.turbolinks
```

Comment:
Is it correct that this means jquery-turbolinks is no longer required? If so, you should drop the gem from `bundler.d/assets.rb` as well.